main.nf

```diff
@@ -125,17 +132,17 @@ if(params.garnett.run == "True"){
             file(ref_marker_genes) from REF_MARKERS
 
         output:
-             file("garnett_output.txt") into GARNETT_OUTPUT
+             file("final_garnett_output.txt") into GARNETT_OUTPUT
 
         """
         RESULTS_DIR=\$PWD
-
+	#pushd $EVAL_WORKFLOWS > dev/null #this is apparently  resume
         nextflow run $EVAL_WORKFLOWS/garnett-eval-workflow/main.nf\
                             -profile cluster\
                             --results_dir \$RESULTS_DIR\
                             --ref_10x_dir ${reference_10X_dir}\
                             --query_10x_dir ${query_10X_dir}\
-                            --marker_genes ${ref_marker_genes}\
+                            --marker_genes ${cv_ref_markes}\
```

Comment:
sorry for being picky, but we want to keep the context of this workflow as general as possible (i.e. it can be run both independently and as part of CV workflow). So, it's better to remove any mentions of cross-validation here

Reply:
Perfect!